src/routes/volume/index.js

```diff
@@ -761,7 +762,7 @@ class Volume extends React.Component {
       v1DataEngineEnabled,
       v2DataEngineEnabled,
       diskTags,
-      backingImages,
+      backingImages: backingImages?.filter(image => hasReadyBackingDisk(image)) || [],
```

Comment:
I've consulted @ChanYiLin if there exists at least one `READY` status backing image disk, user is able to create volume from that backing image and attach to node.

<img width="573" alt="image" src="https://github.com/longhorn/longhorn-ui/assets/5744158/8f4a2c95-5b03-4c5c-bcd7-66d8a59979e4">